### PR TITLE
a2

### PR DESCRIPTION
**Pull request by BRomanchuk**

### CODE REVIEW

**Comment by KyloRen1** [comment]
@BRomanchuk please remove unnecessary files.

**Comment by BRomanchuk** [comment]
> @BRomanchuk please remove unnecessary files.

done

**Comment by KyloRen1** [comment]
@BRomanchuk there are cache files still left

**Comment by BRomanchuk** [comment]
> @BRomanchuk there are cache files still left

and now?

**Comment by KyloRen1** [comment]
@BRomanchuk good

File: BRomanchuk/a2/word2vec.py [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+    """
+
+    ### YOUR CODE HERE (~6-8 Lines)
+    # use pdf notation of U, v_c and u_o
+    U, v_c = outsideVectors, centerWordVec
+    # calculate dot product of U and v_c
+    U_dot_v = np.matmul(U, v_c)
+    # calculate loss and its gradient
+    loss = - U_dot_v[outsideWordIdx] + np.log(np.sum(np.exp(U_dot_v)))
+    gradCenterVec = - U[outsideWordIdx] + np.matmul(np.exp(U_dot_v).T, U).T / np.sum(np.exp(U_dot_v))
```
Comment (KyloRen1) [inline BRomanchuk/a2/word2vec.py:69]: consider using softmax instead of multiple exponents

**Review by KyloRen1 — APPROVED** [review]
LGTM. Pay attention to the comment above
